src/current/cockroachcloud/costs.md

```diff
@@ -256,7 +256,7 @@ In CockroachDB {{ site.data.products.basic }}, Change Data Capture (CDC) cost is
 
 <section class="filter-content" markdown="1" data-scope="standard">
 
-In CockroachDB {{ site.data.products.standard }} and {{ site.data.products.advanced }}, CDC is billed monthly based on usage, determined by the total GiB-Month watched across all of a cluster’s changefeeds. The per-GiB unit price is tiered, based on the total watched: Less than 5 GiB-Month, 5 to 100 GiB-Month, 100 to 250 GiB-Month, 250 to 500 GiB-Month, or 500 GiB-Month and higher.
+In CockroachDB {{ site.data.products.standard }}, CDC is billed monthly based on usage, determined by the total GiB-Month watched across all of a cluster’s changefeeds. The per-GiB unit price is tiered, based on the total watched: Less than 5 GiB-Month, 5 to 100 GiB-Month, 100 to 250 GiB-Month, 250 to 500 GiB-Month, or 500 GiB-Month and higher.
```

Comment:
I think you need to either explode the acronym in all tabs, or explode it in the para before the tabs.